site/content/en/docs/commands/start.md

```diff
@@ -89,6 +89,7 @@ minikube start [flags]
       --registry-mirror strings           Registry mirrors to pass to the Docker daemon
       --service-cluster-ip-range string   The CIDR to be used for service cluster IPs. (default "10.96.0.0/12")
       --trace string                      Send trace events. Options include: [gcp]
+      --user string                       Sets who will be logged as executing the command, will use os user if none provided
```

Comment:
lets make a better description.
how about, 

"specify the user doing an operation. useful for auditing operations done by 3rd party tools.  defaults to the operating system username"

Reply:
Updated